ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1119,6 +1189,16 @@ export const RuntimePanel = fp.flow(
                     }}
                     disabled={disableControls}
                     diskSize={diskSize}/>
+                {
+                  enableGpu && selectedCompute === ComputeType.Standard &&
+                  <GpuConfigSelector
+                      disabled={disableControls}
```

Comment:
I saw terra disables this if in Dataproc mode? 
https://github.com/DataBiosphere/terra-ui/blob/dev/src/components/ComputeModal.js#L712
What happened to us if we are in Dataproc mode

Reply:
The GpuConfigSelector won't exist in Dataproc panel. The user can't use it then he is in Dataproc mode. But I agree it's also safe to disable it here.